global.json

```diff
@@ -11,7 +11,7 @@
     }
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.19260.1",
+    "Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.19267.3",
```

Comment:
Does this work correctly when you're not updating eng/Version.Details.xml ?

Reply:
Yes, eng/version.details.xml is just used for darc to understand / track dependencies.  global.json is what the repo actually uses for these things.